status-im/status-react
# [#2998] Allow both release and develop version to coexist

fixes #2998

### Summary:
Change app id for debug build type so both degub and release build type can coexist. If build debug version with dev clojure profile(which is default profile for dev) and release version with prod clojure profile(witch is also default) then function `status.im.core.debug?` will return `true` for degub build and `false` for release build.

### Steps to test:
- Install release version
- Install debug version
- Check that two apps exist: Status and Status Debug.

![photo_2018-04-26_13-30-22](https://user-images.githubusercontent.com/5045098/39287362-ef70f8c8-4955-11e8-9633-4fbe4129d4e1.jpg)
![photo_2018-04-26_13-30-18](https://user-images.githubusercontent.com/5045098/39287364-f060c722-4955-11e8-9d55-a261f8984960.jpg)

status: ready
